index.js

```diff
@@ -39,6 +39,10 @@ var Wrapper = exports.Wrapper = function Wrapper(fn, context) {
 		throw new Error('AsyncDI Wrapper must be initialised with a function');
 	}
 	
+	this.options = _.defaults({}, opts, {
+		callbackParamName: "callback"
```

Comment:
how about just `callback` or `callbackName`? I think it's pretty clear it's a mapping, would prefer brevity if you agree.